skimage/measure/_regionprops.py

```diff
@@ -358,22 +358,73 @@ def __eq__(self, other):
         return True
 
 
+def to_dict(regions, want=['label', 'bbox'], separator='-', cancel_auto=False):
+    """Convert image regions properties into a dictionary
+
+    Parameters
+    ----------
+    regions : (N,) list
+        _RegionProperties
+    want : list, optional
+        str of a property the user wants to include
+    separator : str, optional
+        used in joining coordinates
+    cancel_auto : bool, optional
+        prevents autmatic adding of label and bbox
+
+
+    Output
+    ------
+    A dictionary with properties in accordance to the wanted properties
+
+    Examples
+    --------
+    >>> from skimage import data, util
+    >>> from skimage.measure import label
+    >>> from pandas.DataFrame import from_dict
+    >>> img = util.img_as_ubyte(data.coins()) > 110
+    >>> label_img = label(img, connectivity=img.ndim)
+    >>> props = regionprops(label_img)
+    >>> out = from_dict(to_dict(props))
```

Comment:
I think you can directly pass a dict to a dataframe, so I think the example should be `props_table = pd.DataFrame(result)`

Also, since we don't want to depend on pandas, you need to add the comment `  # doctest: +SKIP` to all the lines that depend on pandas. You should put them at the very end. Here's how I would structure this:

```python
>>> from skimage import data, util, measure
>>> image = data.coins() > 110
>>> label_image = measure.label(image, connectivity=image.ndim)
>>> propslist = regionprops(label_image, image)
>>> props = to_dict(propslist, want=['label', 'inertia_tensor', 'inertia_tensor_eigvals'])
>>> props
<output here>

The resulting dictionary can be directly passed to pandas, if installed, to
obtain a clean DataFrame::

>>> import pandas as pd  # doctest: +SKIP
>>> data = pd.DataFrame(props)  # doctest: +SKIP
>>> data.head()  # doctest: +SKIP
<output of data.head()>
```